taropsbt/interface.go

```diff
@@ -78,6 +78,11 @@ type VPacket struct {
 	// and Interactive being set to true.
 	Outputs []*VOutput
 
+	// StaleAssets is a list of all stale assets. A stale asset is a
+	// non-input assets which require re-signing and re-anchoring as a
+	// result of the spend.
+	StaleAssets []*asset.Asset
```

Comment:
Don't think we want this to live here, as these are sort of "implicit spends". The other party doesn't need to know about these.

Reply:
Removed :+1: